packages/cli/src/commands/init.ts

```diff
@@ -123,13 +115,15 @@ export const initCommand = async (options: InitCommandOptions) => {
       resolvedPath,
       routeDir,
       resolvedOptions,
+      isTypescriptProject,
```

Comment:
Needed to add another argument to let the "generator" know the kind(ts/js) of project we are dealing with.